usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -123,7 +123,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/idvs/funding/](/api/v2/idvs/funding/)|POST| Returns File C funding records associated with an IDV |
 |[/api/v2/idvs/funding_rollup/](/api/v2/idvs/funding_rollup/)|POST| Returns aggregated count of awarding agencies, federal accounts, and total transaction obligated amount for all contracts under an IDV |
 |[/api/v2/recipient/](/api/v2/recipient/)|POST| Returns a list of recipients in USAspending DB |
-|[/api/v2/recipient/children/<DUNS\>/](/api/v2/recipient/children/006928857/)|GET| Returns recipient details based on DUNS number |
+|[/api/v2/recipient/children/<DUNS_OR_UEI\>/](/api/v2/recipient/children/006928857/)|GET| Returns recipient details based on DUNS number |
```

Comment:
`Returns recipient details based on DUNS number` should get updated to include `DUNS number or UEI` as well

Reply:
Fixed